bin/populate_env

```diff
@@ -12,6 +12,7 @@ search_url="${REACT_APP_HELX_SEARCH_URL}"
 brand_name="${REACT_APP_UI_BRAND_NAME}"
 tranql_enabled="${REACT_APP_TRANQL_ENABLED:-false}"
 analytics="${REACT_APP_ANALYTICS:-0c39f8410fd548bfa1976957d0248289}"
+hidden_support_sections="${REACT_APP_HIDDEN_SUPPORT_SECTIONS}"
```

Comment:
Is there a good default value to set for this?

Reply:
hmm, I think the default is none? By default, all sections on /support page should be there. The initial goal for this pr is to hide the community section on bdc.

Reply:
ok, as long as the UI is tested with an empty string here and works as expected I say 👍🏻

Reply:
yep, an empty string will just leave all sections there. Going to merge. Thanks for the help Carl!